index.js

```diff
@@ -431,15 +446,24 @@ const fetch = () => {
           }
         }
 
-        dashboard.log([
-          `Lowest fares for an outbound flight is currently \€${[lowestOutboundFare, outboundFareDiffString].filter(i => i).join(" ")}`,
-          `Lowest fares for a return flight is currently \€${[lowestReturnFare, returnFareDiffString].filter(i => i).join(" ")}`
-        ])
-
-        dashboard.plot({
-          outbound: lowestOutboundFare,
-          return: lowestReturnFare
-        })
+        if(oneWay){
+          dashboard.log([
```

Comment:
to reduce duplication here it would be preferable to create two arrays `fareStrings` and `fares` and have the outbound details be pushed if `!oneWay`, then those two arrays can just be passed to `plot` and `log`